src/bfs.py

```diff
@@ -0,0 +1,58 @@
+from collections import deque
+import numpy as np
+ 
+def isValid(vis, floor, row, col):
+    #Cell out of bounds
+    if (row < 0 or col < 0 or row >= 70 or col >= 102 or floor < 0 or floor >= 5):
```

Comment:
would it be possible not to hardcode values that work only for our specific images?

Reply:
done